submissions/AndriiZahumnyi/html-css-popup/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>HTML & CSS practice: Hooli-style Popup</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="main_style.css">
+</head>
+<body>
+  <header class="header">
+    <nav class="head_nav">
+      <div class="container">
+        <ul class="main_menu">
+          <li>Mail</li>
+          <li>Images</li>
+          <li>
+            <input id="check01" type="checkbox" name="menu" />
+            <label for="check01"><img class="popup_png" src="/popup-icons/popup-button.png" alt="popup button"
+                tabindex="1"></label>
+            <div class="sub_menu">
+              <div class="sub_container">
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/calendar.png" alt="calendar icon">
+                    <p>Calendar</p>
+                  </div>
+                </a>
+                <a href="#">
```

Comment:
Wrap your block .sub_container in `ul` | `ol` instead `div`, after this wrap your links in `li`.
We're practicing here for [ally](https://medium.com/web-standards/pragmatic-a11y-rules-b16dd2f77685), so don't forget about semantic.